force-app/main/userCommunity/lwc/hot_home/hot_home.js

```diff
@@ -7,35 +7,42 @@ import checkAssignedPermissionSet from '@salesforce/apex/HOT_Utility.checkAssign
 import checkAssignedPermissionSetGroup from '@salesforce/apex/HOT_Utility.checkAssignedPermissionSetGroup';
 
 export default class Hot_home extends NavigationMixin(LightningElement) {
-    @track error;
-
     @track name;
     @wire(getRecord, {
         recordId: USER_ID,
         fields: [NAME_FIELD]
     })
-    wireuser({ error, data }) {
-        if (error) {
-            this.error = error;
-        } else if (data) {
+    wireuser({ data }) {
+        if (data) {
             this.name = data.fields.FirstName.value;
         }
     }
 
+    @track pageLinks = {};
```

Comment:
vil den ikke tryne om du ikke har med verdiene fra før?
altså at keys er deklarert?

Reply:
Fungerer uten verdier faktisk

Reply:
Tror bare den whiner på det om man prøver å sette en attributt på et objekt i en parent component f. eks. For da har man bare read-only rights til objektet elns. 🤷